src/main/java/me/eigenraven/lwjgl3ify/core/Lwjgl3ifyCoremod.java

```diff
@@ -72,49 +72,22 @@ public String getMixinConfig() {
 
     @Override
     public List<String> getMixins(Set<String> loadedCoreMods) {
-
-        final boolean hasFastcraft = loadedCoreMods.contains("fastcraft.Tweaker");
-        final boolean hasOptifine = loadedCoreMods.contains("optifine.OptiFineForgeTweaker");
-        List<String> mixins = new ArrayList<>(8);
-        // FML Java 9+ compatibility patches
-        mixins.add("fml.ItemStackHolderRef");
-        mixins.add("fml.JarDiscoverer");
-        mixins.add("fml.ObjectHolderRef");
-        mixins.add("fml.ObjectHolderRegistry");
         if (FMLLaunchHandler.side()
             .isClient()) {
-            // Improved KeyBinding handling to handle dead keys
-            mixins.add("game.MixinMinecraftKeyBinding");
-
-            // Adds the borderless mode
-            mixins.add("game.MixinBorderlessWindow");
-
-            // STB replacements for vanilla functions
-            if (Config.MIXIN_STBI_TEXTURE_LOADING) {
-                LOGGER.info("Enabling STB texture loading mixin");
-                mixins.add("game.MixinTextureAtlasSprite");
-                mixins.add("game.MixinTextureMap");
-            } else {
-                LOGGER.info("Disabling STB texture loading mixin");
-            }
-
+            final boolean hasFastcraft = loadedCoreMods.contains("fastcraft.Tweaker");
+            final boolean hasOptifine = loadedCoreMods.contains("optifine.OptiFineForgeTweaker");
             final boolean fcBugFixedByOF = isFastcraftVersion1_25();
             final boolean fcBugTriggered = hasFastcraft && !(hasOptifine && fcBugFixedByOF);
             if (fcBugTriggered && !Config.MIXIN_STBI_IGNORE_FASTCRAFT) {
                 LOGGER.error(
-                    "Not using STB stiching mixins because FastCraft is installed to prevent rapidly flashing screen. Remove FastCraft or "
+                    "Not using STB stitching mixins because FastCraft is installed to prevent rapidly flashing screen. Remove FastCraft or "
                         + (!fcBugFixedByOF ? "update to FastCraft 1.25 and " : "")
                         + "add OptiFine to enable these performance-improving patches.");
-            } else {
-                if (Config.MIXIN_STBI_TEXTURE_STICHING) {
-                    LOGGER.info("Enabling STB texture stitching mixin");
-                    mixins.add("game.MixinStitcher");
-                } else {
-                    LOGGER.info("Disabling STB texture stitching mixin");
-                }
+                Config.MIXIN_STBI_TEXTURE_STITCHING = false;
```

Comment:
Instead of mutating the config here (which will perma-disable the mixin), add it to the ApplyIf conditions in the mixins enum

Reply:
I was afraid of that. Should I move `isFastCraftVersion1_25` into the `Mixins` class or just change the visibility?

Reply:
Let's try just increasing visibility

Reply:
Should be fixed